support-workers/src/main/scala/com/gu/support/workers/lambdas/PreparePaymentMethodForReuse.scala

```diff
@@ -72,7 +72,8 @@ class PreparePaymentMethodForReuse(servicesProvider: ServiceProvider = ServicePr
           creditCardExpirationMonth = cardResponse.creditCardExpirationMonth,
           creditCardExpirationYear = cardResponse.creditCardExpirationYear,
           creditCardType = cardResponse.creditCardType,
-          paymentGateway = paymentGateway
+          paymentGateway = paymentGateway,
+          stripePaymentType = None
```

Comment:
this info has presumably come from zuora, so we do not have the `stripePaymentType`